core/test/computed/metrics/responsiveness-test.js

```diff
@@ -237,28 +237,6 @@ describe('Metric: Responsiveness', () => {
       .rejects.toThrow(`unexpected responsiveness interactionType 'brainWave'`);
   });
 
-  it('returns a fallback timing event if provided with the old trace event format', async () => {
```

Comment:
nit: could we verify the error is thrown here in addition to/instead of in `interaction-to-next-paint-test.js`? Just cuz the logic to throw the error lives in `responsiveness.js`.